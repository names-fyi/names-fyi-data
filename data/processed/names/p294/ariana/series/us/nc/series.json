[{"y":1988,"t":5,"m":0,"f":5,"mr":null,"fr":803,"mp":0,"fp":0.00013042570951585976},{"y":1989,"t":18,"m":0,"f":18,"mr":null,"fr":330,"mp":0,"fp":0.0004424234976035394},{"y":1990,"t":10,"m":0,"f":10,"mr":null,"fr":510,"mp":0,"fp":0.0002447620912473076},{"y":1991,"t":17,"m":0,"f":17,"mr":null,"fr":348,"mp":0,"fp":0.0004265676360625298},{"y":1992,"t":18,"m":0,"f":18,"mr":null,"fr":324,"mp":0,"fp":0.00045186393874733274},{"y":1993,"t":18,"m":0,"f":18,"mr":null,"fr":325,"mp":0,"fp":0.0004690309299841051},{"y":1994,"t":32,"m":0,"f":32,"mr":null,"fr":220,"mp":0,"fp":0.0008387942332896461},{"y":1995,"t":38,"m":0,"f":38,"mr":null,"fr":196,"mp":0,"fp":0.0009932303510285163},{"y":1996,"t":29,"m":0,"f":29,"mr":null,"fr":246,"mp":0,"fp":0.0007369758576874206},{"y":1997,"t":26,"m":0,"f":26,"mr":null,"fr":279,"mp":0,"fp":0.0006491236830279124},{"y":1998,"t":45,"m":0,"f":45,"mr":null,"fr":195,"mp":0,"fp":0.0010691629641949203},{"y":1999,"t":39,"m":0,"f":39,"mr":null,"fr":216,"mp":0,"fp":0.0009177982255900972},{"y":2000,"t":58,"m":0,"f":58,"mr":null,"fr":165,"mp":0,"fp":0.0012864018453212678},{"y":2001,"t":72,"m":0,"f":72,"mr":null,"fr":126,"mp":0,"fp":0.001627891202604626},{"y":2002,"t":55,"m":0,"f":55,"mr":null,"fr":170,"mp":0,"fp":0.0012661433274246644},{"y":2003,"t":90,"m":0,"f":90,"mr":null,"fr":103,"mp":0,"fp":0.002069060646466504},{"y":2004,"t":85,"m":0,"f":85,"mr":null,"fr":104,"mp":0,"fp":0.0019124760940488245},{"y":2005,"t":76,"m":0,"f":76,"mr":null,"fr":125,"mp":0,"fp":0.0016743776162150253},{"y":2006,"t":82,"m":0,"f":82,"mr":null,"fr":121,"mp":0,"fp":0.001741309379711622},{"y":2007,"t":95,"m":0,"f":95,"mr":null,"fr":107,"mp":0,"fp":0.001975051975051975},{"y":2008,"t":98,"m":0,"f":98,"mr":null,"fr":102,"mp":0,"fp":0.0020386511618231368},{"y":2009,"t":107,"m":0,"f":107,"mr":null,"fr":88,"mp":0,"fp":0.002305835703818636},{"y":2010,"t":99,"m":0,"f":99,"mr":null,"fr":94,"mp":0,"fp":0.0022033294756520967},{"y":2011,"t":99,"m":0,"f":99,"mr":null,"fr":88,"mp":0,"fp":0.002242761995378551},{"y":2012,"t":104,"m":0,"f":104,"mr":null,"fr":89,"mp":0,"fp":0.0023672949103159427},{"y":2013,"t":137,"m":0,"f":137,"mr":null,"fr":62,"mp":0,"fp":0.003093528428848846},{"y":2014,"t":162,"m":0,"f":162,"mr":null,"fr":48,"mp":0,"fp":0.0035916991841078395},{"y":2015,"t":138,"m":0,"f":138,"mr":null,"fr":62,"mp":0,"fp":0.0030411212481819385},{"y":2016,"t":137,"m":0,"f":137,"mr":null,"fr":61,"mp":0,"fp":0.0030252175065141546},{"y":2017,"t":108,"m":0,"f":108,"mr":null,"fr":86,"mp":0,"fp":0.0023817925194071984},{"y":2018,"t":97,"m":0,"f":97,"mr":null,"fr":95,"mp":0,"fp":0.0021570894858566093},{"y":2019,"t":94,"m":0,"f":94,"mr":null,"fr":99,"mp":0,"fp":0.0020971376302344777},{"y":2020,"t":110,"m":0,"f":110,"mr":null,"fr":74,"mp":0,"fp":0.0025248467877062915},{"y":2021,"t":93,"m":0,"f":93,"mr":null,"fr":98,"mp":0,"fp":0.002068551346782624},{"y":2022,"t":87,"m":0,"f":87,"mr":null,"fr":106,"mp":0,"fp":0.0019291321122887933}]