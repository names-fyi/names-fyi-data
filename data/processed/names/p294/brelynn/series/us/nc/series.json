[{"y":2016,"t":7,"m":0,"f":7,"mr":null,"fr":1114,"mp":0,"fp":0.00015457315726714657},{"y":2021,"t":5,"m":0,"f":5,"mr":null,"fr":1449,"mp":0,"fp":0.00011121243799906581}]