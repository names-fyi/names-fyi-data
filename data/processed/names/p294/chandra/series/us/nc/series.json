[{"y":1956,"t":7,"m":0,"f":7,"mr":null,"fr":557,"mp":0,"fp":0.0001352029976436049},{"y":1957,"t":5,"m":0,"f":5,"mr":null,"fr":676,"mp":0,"fp":0.00009876153040867522},{"y":1958,"t":10,"m":0,"f":10,"mr":null,"fr":463,"mp":0,"fp":0.00020317769921573408},{"y":1959,"t":14,"m":0,"f":14,"mr":null,"fr":392,"mp":0,"fp":0.0002853299637223332},{"y":1961,"t":7,"m":0,"f":7,"mr":null,"fr":574,"mp":0,"fp":0.00014353380221042057},{"y":1962,"t":5,"m":0,"f":5,"mr":null,"fr":673,"mp":0,"fp":0.0001048679712242287},{"y":1965,"t":7,"m":0,"f":7,"mr":null,"fr":547,"mp":0,"fp":0.00016745610257882398},{"y":1966,"t":8,"m":0,"f":8,"mr":null,"fr":500,"mp":0,"fp":0.00020158242201280047},{"y":1967,"t":15,"m":0,"f":15,"mr":null,"fr":337,"mp":0,"fp":0.0003806140573458513},{"y":1968,"t":14,"m":0,"f":14,"mr":null,"fr":358,"mp":0,"fp":0.000356361044646948},{"y":1969,"t":15,"m":0,"f":15,"mr":null,"fr":347,"mp":0,"fp":0.00037655328228944395},{"y":1970,"t":24,"m":0,"f":24,"mr":null,"fr":279,"mp":0,"fp":0.0005729427773401132},{"y":1971,"t":21,"m":0,"f":21,"mr":null,"fr":287,"mp":0,"fp":0.0005215318134406199},{"y":1972,"t":21,"m":0,"f":21,"mr":null,"fr":282,"mp":0,"fp":0.0005721759032205329},{"y":1973,"t":23,"m":0,"f":23,"mr":null,"fr":270,"mp":0,"fp":0.0006595171187704306},{"y":1974,"t":24,"m":0,"f":24,"mr":null,"fr":255,"mp":0,"fp":0.000709177944565924},{"y":1975,"t":23,"m":0,"f":23,"mr":null,"fr":264,"mp":0,"fp":0.0007110396636473243},{"y":1976,"t":33,"m":0,"f":33,"mr":null,"fr":203,"mp":0,"fp":0.0010316368638239339},{"y":1977,"t":14,"m":0,"f":14,"mr":null,"fr":358,"mp":0,"fp":0.00042331881954523463},{"y":1978,"t":24,"m":0,"f":24,"mr":null,"fr":251,"mp":0,"fp":0.0007413585395236772},{"y":1979,"t":23,"m":0,"f":23,"mr":null,"fr":254,"mp":0,"fp":0.0006998113551999026},{"y":1980,"t":16,"m":0,"f":16,"mr":null,"fr":311,"mp":0,"fp":0.00048118853568313735},{"y":1981,"t":12,"m":0,"f":12,"mr":null,"fr":382,"mp":0,"fp":0.0003620892550013578},{"y":1982,"t":7,"m":0,"f":7,"mr":null,"fr":571,"mp":0,"fp":0.00020466639377814163},{"y":1983,"t":8,"m":0,"f":8,"mr":null,"fr":526,"mp":0,"fp":0.000239736290080911},{"y":1984,"t":8,"m":0,"f":8,"mr":null,"fr":504,"mp":0,"fp":0.00023469357819696658},{"y":1985,"t":19,"m":0,"f":19,"mr":null,"fr":288,"mp":0,"fp":0.0005344434755703074},{"y":1986,"t":11,"m":0,"f":11,"mr":null,"fr":432,"mp":0,"fp":0.00030935373193092977},{"y":1987,"t":5,"m":0,"f":5,"mr":null,"fr":784,"mp":0,"fp":0.0001341849605496216},{"y":1988,"t":5,"m":0,"f":5,"mr":null,"fr":815,"mp":0,"fp":0.00013042570951585976},{"y":1989,"t":5,"m":0,"f":5,"mr":null,"fr":861,"mp":0,"fp":0.00012289541600098315},{"y":1990,"t":8,"m":0,"f":8,"mr":null,"fr":614,"mp":0,"fp":0.00019580967299784609},{"y":1991,"t":8,"m":0,"f":8,"mr":null,"fr":623,"mp":0,"fp":0.0002007377110882493},{"y":1992,"t":5,"m":0,"f":5,"mr":null,"fr":926,"mp":0,"fp":0.000125517760763148},{"y":1995,"t":6,"m":0,"f":6,"mr":null,"fr":791,"mp":0,"fp":0.0001568258448992394}]