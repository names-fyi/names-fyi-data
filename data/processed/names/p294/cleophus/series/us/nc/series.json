[{"y":1923,"t":7,"m":7,"f":0,"mr":443,"fr":null,"mp":0.00020971269360975465,"fp":0},{"y":1926,"t":5,"m":5,"f":0,"mr":533,"fr":null,"mp":0.00014715404084996174,"fp":0},{"y":1930,"t":7,"m":7,"f":0,"mr":431,"fr":null,"mp":0.0002100840336134454,"fp":0},{"y":1945,"t":5,"m":5,"f":0,"mr":504,"fr":null,"mp":0.00012114163880408974,"fp":0}]