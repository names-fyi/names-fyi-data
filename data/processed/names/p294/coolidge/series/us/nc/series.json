[{"y":1923,"t":6,"m":6,"f":0,"mr":498,"fr":null,"mp":0.00017975373737978969,"fp":0},{"y":1924,"t":13,"m":13,"f":0,"mr":295,"fr":null,"mp":0.0003732736095558044,"fp":0},{"y":1925,"t":11,"m":11,"f":0,"mr":318,"fr":null,"mp":0.00032271313735844626,"fp":0},{"y":1926,"t":5,"m":5,"f":0,"mr":535,"fr":null,"mp":0.00014715404084996174,"fp":0},{"y":1927,"t":5,"m":5,"f":0,"mr":570,"fr":null,"mp":0.00014237712853807163,"fp":0},{"y":1929,"t":6,"m":6,"f":0,"mr":472,"fr":null,"mp":0.00018088088993397847,"fp":0}]