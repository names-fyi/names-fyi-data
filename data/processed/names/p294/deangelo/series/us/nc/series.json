[{"y":1974,"t":6,"m":6,"f":0,"mr":511,"fr":null,"mp":0.0001545674687000876,"fp":0},{"y":1979,"t":7,"m":7,"f":0,"mr":465,"fr":null,"mp":0.00018634863166861889,"fp":0},{"y":1982,"t":7,"m":7,"f":0,"mr":458,"fr":null,"mp":0.00018191740949608878,"fp":0},{"y":1983,"t":10,"m":10,"f":0,"mr":376,"fr":null,"mp":0.0002632687447346251,"fp":0},{"y":1984,"t":8,"m":8,"f":0,"mr":419,"fr":null,"mp":0.00020462974804962272,"fp":0},{"y":1985,"t":6,"m":6,"f":0,"mr":534,"fr":null,"mp":0.00014806040864672785,"fp":0},{"y":1986,"t":7,"m":7,"f":0,"mr":506,"fr":null,"mp":0.00017121193591781826,"fp":0},{"y":1987,"t":11,"m":11,"f":0,"mr":373,"fr":null,"mp":0.0002649453249193121,"fp":0},{"y":1988,"t":5,"m":5,"f":0,"mr":653,"fr":null,"mp":0.0001149213937666636,"fp":0},{"y":1989,"t":13,"m":13,"f":0,"mr":380,"fr":null,"mp":0.00028108108108108106,"fp":0},{"y":1990,"t":16,"m":16,"f":0,"mr":347,"fr":null,"mp":0.0003367782946389105,"fp":0},{"y":1991,"t":17,"m":17,"f":0,"mr":339,"fr":null,"mp":0.00036919602136993443,"fp":0},{"y":1992,"t":11,"m":11,"f":0,"mr":463,"fr":null,"mp":0.00023699745766363596,"fp":0},{"y":1993,"t":14,"m":14,"f":0,"mr":398,"fr":null,"mp":0.0003121168208672389,"fp":0},{"y":1994,"t":9,"m":9,"f":0,"mr":519,"fr":null,"mp":0.00020119824733971206,"fp":0},{"y":1995,"t":8,"m":8,"f":0,"mr":594,"fr":null,"mp":0.00017853954650955185,"fp":0},{"y":1996,"t":9,"m":9,"f":0,"mr":564,"fr":null,"mp":0.00019615963034807436,"fp":0},{"y":1997,"t":17,"m":17,"f":0,"mr":374,"fr":null,"mp":0.0003616636528028933,"fp":0},{"y":1998,"t":10,"m":10,"f":0,"mr":569,"fr":null,"mp":0.00020611756946162092,"fp":0},{"y":1999,"t":9,"m":9,"f":0,"mr":621,"fr":null,"mp":0.00018286736020806243,"fp":0},{"y":2000,"t":12,"m":12,"f":0,"mr":542,"fr":null,"mp":0.00023198716337695982,"fp":0},{"y":2001,"t":6,"m":6,"f":0,"mr":825,"fr":null,"mp":0.00011885189073549512,"fp":0},{"y":2002,"t":5,"m":5,"f":0,"mr":941,"fr":null,"mp":0.00009948269001193792,"fp":0},{"y":2005,"t":5,"m":5,"f":0,"mr":1009,"fr":null,"mp":0.00009558584564797645,"fp":0},{"y":2006,"t":8,"m":8,"f":0,"mr":790,"fr":null,"mp":0.00014685096462727388,"fp":0},{"y":2007,"t":7,"m":7,"f":0,"mr":886,"fr":null,"mp":0.00012460837368271147,"fp":0},{"y":2008,"t":8,"m":8,"f":0,"mr":823,"fr":null,"mp":0.0001436085231658499,"fp":0},{"y":2009,"t":12,"m":12,"f":0,"mr":637,"fr":null,"mp":0.0002229240200631618,"fp":0},{"y":2011,"t":8,"m":8,"f":0,"mr":830,"fr":null,"mp":0.00015726670467278695,"fp":0},{"y":2012,"t":5,"m":5,"f":0,"mr":1126,"fr":null,"mp":0.00009892762454987931,"fp":0},{"y":2013,"t":6,"m":6,"f":0,"mr":966,"fr":null,"mp":0.00011907833369718379,"fp":0},{"y":2014,"t":8,"m":8,"f":0,"mr":817,"fr":null,"mp":0.00015482572429409146,"fp":0},{"y":2015,"t":8,"m":8,"f":0,"mr":852,"fr":null,"mp":0.000153932000538762,"fp":0},{"y":2016,"t":5,"m":5,"f":0,"mr":1172,"fr":null,"mp":0.00009626306771144182,"fp":0},{"y":2017,"t":6,"m":6,"f":0,"mr":1064,"fr":null,"mp":0.00011699783554004251,"fp":0},{"y":2019,"t":7,"m":7,"f":0,"mr":995,"fr":null,"mp":0.00013974566289353378,"fp":0},{"y":2020,"t":6,"m":6,"f":0,"mr":1095,"fr":null,"mp":0.00012281491791869652,"fp":0},{"y":2021,"t":5,"m":5,"f":0,"mr":1306,"fr":null,"mp":0.00009816818172893802,"fp":0}]