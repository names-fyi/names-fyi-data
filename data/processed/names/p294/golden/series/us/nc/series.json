[{"y":1913,"t":5,"m":5,"f":0,"mr":337,"fr":null,"mp":0.0003507787287778869,"fp":0},{"y":1917,"t":6,"m":6,"f":0,"mr":458,"fr":null,"mp":0.000226928895612708,"fp":0},{"y":1919,"t":6,"m":6,"f":0,"mr":494,"fr":null,"mp":0.00019579689335595874,"fp":0},{"y":1922,"t":5,"m":5,"f":0,"mr":590,"fr":null,"mp":0.0001471323897242739,"fp":0},{"y":1923,"t":5,"m":5,"f":0,"mr":575,"fr":null,"mp":0.00014979478114982475,"fp":0},{"y":1930,"t":5,"m":5,"f":0,"mr":548,"fr":null,"mp":0.00015006002400960383,"fp":0},{"y":1932,"t":5,"m":5,"f":0,"mr":531,"fr":null,"mp":0.0001470631489161446,"fp":0}]